packages/acceptance-tests/pkg-tests-core/sources/utils/tests.ts

```diff
@@ -233,7 +233,7 @@ export const startPackageServer = (): Promise<string> => {
                 [version as string]: Object.assign({}, packageVersionEntry!.packageJson, {
                   dist: {
                     shasum: await getPackageArchiveHash(name, version),
-                    tarball: localName === `unconventional-tarball`
+                    tarball: (localName === `unconventional-tarball` || localName === `private-unconventional-tarball`)
```

Comment:
I was considering something like this (along the same lines of ``localName.startsWith(`private`)``):
```suggestion
                    tarball: (localName.includes(`unconventional-tarball`))
```

Reply:
It's ok for now - if we need to add more private packages we will just move the whitelist into a set 👍